src/renderer/components/editors.tsx

```diff
@@ -266,32 +261,6 @@ export class Editors extends React.Component<EditorsProps, EditorsState> {
     this.props.appState.editorMosaic.mosaicArrangement = currentNode;
   }
 
-  /**
-   * Loads monaco. If it's already loaded, it'll just set it on the current state.
-   * We're doing things a bit roundabout to ensure that we're not overloading the
```

Comment:
is there potential this happens with this refactor?

Reply:
answered in 1:1